docs/integrations/sources/confluence.md

```diff
@@ -49,4 +49,5 @@ Follow [this article](https://support.atlassian.com/atlassian-account/docs/manag
 
 | Version | Date | Pull Request | Subject |
 | :--- | :--- | :--- | :--- |
+| 0.1.1 | 2022-01-31 | [9831](https://github.com/airbytehq/airbyte/pull/9831) | chore: added spec to cache and added to source definition file |
```

Comment:
Could you rephrase it to something less technical, easier to understand for connector users?